osconfig_tests/test_suites/guestpolicies/guest_policies.go

```diff
@@ -0,0 +1,223 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// Package guestpolicies GuestPolicy osconfig agent tests.
+
+package guestpolicies
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"path"
+	"regexp"
+	"sync"
+	"time"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/go/e2e_test_utils/junitxml"
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/compute"
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/config"
+	gcpclients "github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/gcp_clients"
+	testconfig "github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/test_config"
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/utils"
+	"github.com/kylelemons/godebug/pretty"
+	computeApi "google.golang.org/api/compute/v1"
+
+	osconfigpb "github.com/GoogleCloudPlatform/osconfig/_internal/gapi-cloud-osconfig-go/google.golang.org/genproto/googleapis/cloud/osconfig/v1alpha2"
+)
+
+var (
+	testSuiteName = "OSPackage"
+)
+
+var (
+	dump = &pretty.Config{IncludeUnexported: true}
+)
+
+const (
+	packageInstallFunction            = "pkginstall"
+	packageRemovalFunction            = "pkgremoval"
+	packageInstallFromNewRepoFunction = "pkgfromnewrepo"
+	packageUpdateFunction             = "pkgupdate"
+	packageNoUpdateFunction           = "pkgnoupdate"
+)
+
+type packageManagementTestSetup struct {
+	image         string
+	name          string
```

Comment:
Here is where you should consider changing this to `guestPolicyId` and adding `vmInstanceId`. It also threw me for a loop since `name` is also used for the label. But I'm guessing that will be changed as soon as the fix is deployed.